docs/src/main/asciidoc/websockets-next-reference.adoc

```diff
@@ -1203,6 +1254,39 @@ quarkus.log.category."io.quarkus.websockets.next.traffic".level=DEBUG <3>
 <2> Set the number of characters of a text message payload which will be logged.
 <3> Enable `DEBUG` level is for the logger `io.quarkus.websockets.next.traffic`.
 
+[[subscribe-or-not-subscribe]]
+== When to subscribe to a `Uni` or `Multi`
+
+`Uni` and `Multi` are lazy types, which means that they do not start processing until they are subscribed to.
+
+When you call a method returning a `Multi` or `Uni`, or receive a `Multi` or `Uni` as a parameter, whether you should subscribe to it depends on the context:
```

Comment:
This sentence does not sound right to me. Maybe a native speaker could take a look... @holly-cummins @dmlloyd?